mne/io/kit/constants.py

```diff
@@ -46,6 +46,7 @@
     50: (2, 1, 1),  # Hanger Type #3
     60: (2, 1, 1),  # Hanger Type #3
     100: (3, 3, 3),  # Low Band Kapper Type
+    120: (3, 3, 3),  # Low Band Kapper Type
```

Comment:
is `Low Band Kapper Type` adapted here?

Reply:
Apparently there is a new control interface but the same filters and amplifiers.